packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -146,7 +147,7 @@ function flattenTree( input = {}, prefix, token ) {
  * @return {Array} An array of style declarations.
  */
 function getStylesDeclarations( blockStyles = {} ) {
-	return reduce(
+	const output = reduce(
 		STYLE_PROPERTY,
 		( declarations, { value, properties }, key ) => {
```

Comment:
I think we should probably ignore the properties with useEngine: true here kind of like https://github.com/WordPress/gutenberg/pull/37978/files#diff-292053c9ec98ebdd73943767587e8093315ad3a2bf01ca3b5352efc48326becdR89

Reply:
Good catch! I've added `useEngine` to the check that bails early in the loop in https://github.com/WordPress/gutenberg/pull/37978/commits/31981a16172a30f8d49d47b55f5a889c8121adc6 — I've also updated the output slightly to match the change to `getCSSRules` [discussed here](https://github.com/WordPress/gutenberg/pull/37978#discussion_r791393936).